src/facade/dragonfly_connection.cc

```diff
@@ -431,6 +431,55 @@ string Connection::GetClientInfo(unsigned thread_id) const {
   return res;
 }
 
+string Connection::GetClientInfo() const {
+  auto res = GetClientInfo(0);
+  auto start = res.find("tid");
+  auto end = res.find("irq");
+  res.erase(start, end - start);
+
+  // The following are dummy fields and users should not rely on those unless
+  // we decide to implement them.
+  // This is only done because the redis pyclient parser for the field "client-info"
+  // for the command ACL LOG
+  // is completely *BROKEN* and it literally hardcodes the expected values.
+  // What is more preposterous is that that it actually doesn't conform to the
+  // actual expected values, since it's missing half of them. That is, even for
+  // redis-server, issuing an ACL LOG command via redis-cli and the pyclient
+  // will return different results! An example of ACL LOG ran with redis-cli is:
+  //
+  // "client-info"
+  // 14) "id=3 addr=127.0.0.1:57275 laddr=127.0.0.1:6379 fd=8 name= age=16
+  //      idle=0 flags=N db=0 sub=0 psub=0 ssub=0 multi=-1 qbuf=48 qbuf-free=16842
+  //      argv-mem=25 multi-mem=0 rbs=1024 rbp=0 obl=0 oll=0 omem=0
+  //      tot-mem=18737 events=r cmd=auth user=default redir=-1 resp=2"
+  //
+  // Meanwhile the parser in the pyclient:
+  //
```

Comment:
It was completely broken. It basically filtered most of the output fields. 

> worked for hundreds of thousands of people I assume its not seriously broken
It worked so well that they refactored all of its code on the next version :stuck_out_tongue: 

> you can just add a comment above strappends below that this if for compatibility with redis-py 4.X
Yes I want to first check if all works with 5.0. But it was a rollercoaster, because my tests would fail because the pyclient could not parse the correct by the protocol output